### DISCUSSION TITLE
Quality is worse than on other YT Music Clients

### DISCUSSION THREAD

**Discussion by TheTrashCrafter** [KRTirtho/spotube#593] [General]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Everything just sounds very flat and far from good.

### Expected Behavior

It should sound just as good as for example InnerTune.

### Steps to reproduce

Just play a song and compare it to InnerTune or something else.

### Operating System

Android

### Spotube version

3.0.0 

### Installation source

GitHub Actions (Nightly Binary)

### Additional information

Thanks for the amazing work you've done, I really like your app, but this makes it unusable for me.

**Comment by KRTirtho** [comment]
Unfortunately, audio optimizations is not in my hand. Spotube matches whatever on YouTube and uses the highest 256 kbps bitrate audio if _"Audio Quality: High"_ is selected and 128 kbps if _"Audio Quality: Low"_ is selected
I don't know if InnerTune does somekind of mixing before hand or uses a different/custom equalizer but Spotube doesn't alter anything like it

**Comment by TheTrashCrafter** [comment]
@KRTirtho I think it'd be weird if InnerTune did that, all InnerTune does is (AFAIK) get the Track from YouTube Music and play it.
